src/FluentResults/FluentResults.csproj

```diff
@@ -30,6 +30,7 @@
   </ItemGroup>
 
   <ItemGroup>
+    <PackageReference Include="Microsoft.Extensions.Logging.Abstractions" Version="2.0.0" />
```

Comment:
Why do you insert such an old version of this package? The newest version is https://www.nuget.org/packages/Microsoft.Extensions.Logging.Abstractions 6.1

Reply:
No reason. Updated.